src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/LanguageServerProjectSystem.cs

```diff
@@ -96,7 +98,10 @@ public async Task OpenSolutionAsync(string solutionFilePath)
 
             // If we don't have a .NET Core SDK on this machine at all, try .NET Framework
             if (!await buildHost.HasUsableMSBuildAsync(solutionFilePath, CancellationToken.None))
-                buildHost = await buildHostProcessManager.GetBuildHostAsync(BuildHostProcessManager.BuildHostProcessKind.NetFramework, CancellationToken.None);
+            {
```

Comment:
as before, would be nice if all the fallback code was in one place.

Reply:
Leaving as is since this is being refactored soon enough.